harvesters/src/main/java/org/fao/geonet/kernel/harvest/HarvestManagerImpl.java

```diff
@@ -79,6 +82,10 @@
 
     private Map<String, AbstractHarvester> hmHarvesters   = new HashMap<String, AbstractHarvester>();
     private Map<String, AbstractHarvester> hmHarvestLookup= new HashMap<String, AbstractHarvester>();
+    
+    private final List<String> summaryHarvesterSettings = Arrays.asList("harvesting","node","site","name","uuid","url","capabUrl","baseUrl","useAccount","ogctype","options","status","info","lastRun");
```

Comment:
It looks like this is used to retrieve harvester params from the settings but in that case, you only put the params for OGC WxS type and you probably have to take care of all other type of harvester params no ? Only node, site name and uuid is common for all types, then parameters are specific. 

See for example:
https://github.com/geonetwork/core-geonetwork/blob/develop/harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/thredds/ThreddsParams.java#L61